app/javascript/modules/listingDetailsAside/ListingDetailsApply.tsx

```diff
@@ -43,14 +43,15 @@ const FcfsBmrSalesHowToApply = ({ listingId }: { listingId: string }) => (
         <li>{t("listings.fcfs.bmrSales.howToApply.step3")}</li>
       </ol>
     </div>
-    <LinkButton
+    <Button
```

Comment:
question: can we use the [ui-seeds button component](https://github.com/bloom-housing/ui-seeds/blob/main/src/actions/Button.tsx)? Using `onClick` to mimic the link-button functionality doesn't seem ideal.